ci/gpu/build.sh

```diff
@@ -59,15 +59,21 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=0.12*" \
-      "statsmodels" \
-      "xgboost=0.90.rapidsdev1"
```

Comment:
I'll make the same recommendation that was made to me when I added the built from source dask and distributed: don't remove the installation here (update the version), so that prerequisites of xgboost are installed and then you can install it from source without worrying about prereqs